gamemodes/terrortown/gamemode/shared/lang/german.lua

```diff
@@ -664,19 +667,19 @@ L.spec_help2 = "Zum Verlassen des Zuschauer-Modus öffne das Menü mit {helpkey}
 -- quotes. That's a Lua thing. Every line break (enter) will show up in-game.
 L.info_popup_innocent = [[Du bist ein unschuldiger Terrorist!
 Aber es gibt Verräter… Wem kannst du trauen, und wem füllst du den Körper mit Blei?
-Gib Acht und arbeite mit anderen zusammen um zu überleben!]]
+Gib acht und arbeite mit anderen zusammen, um zu überleben!]]
```

Comment:
That's wrong. "Acht" is a substantive in that case (it isn't written together). Ref.: https://www.duden.de/suchen/dudenonline/Gib%20acht

Reply:
Most of the spelling and grammar changes were proposed by word, Ill change it